packages/components/src/components/tabs/tab.jsx

```diff
@@ -35,6 +36,8 @@ const Tab = ({
         'dc-tabs__item--header-fit-content': header_fit_content,
         'dc-tabs__item--is-hidden': is_label_hidden,
         [`dc-tabs__item--${className}`]: className,
+        'dc-tabs__active:after': is_scrollable,
+        'dc-tabs__active--is-p2p': is_scrollable,
```

Comment:
What if another app passes the `is_scrollable` prop? In that case it'll be considered `--is-p2p` which doesn't look correct.